src/dataServices/dynamoDbBundleService.test.ts

```diff
@@ -9,16 +9,27 @@ import AWSMock from 'aws-sdk-mock';
 import { QueryInput, TransactWriteItemsInput } from 'aws-sdk/clients/dynamodb';
 import * as AWS from 'aws-sdk';
 import { BundleResponse, BatchReadWriteRequest } from 'fhir-works-on-aws-interface';
+import { SinonSandbox } from 'sinon';

```

Comment:
Why?

sinon > 5.0.0 defaults to behave as a sandbox, so we may not need to create sandboxes explicitly

https://sinonjs.org/releases/latest/sandbox/

** btw, not sure why we are using sinon instead of Jest mock functions to begin with.

Reply:
In this case, if we don't use `sandbox`, we'll get this error. 
`TypeError: Attempted to wrap getMostRecentResource which is already wrapped`
In this test file we call `runUpdateTest` twice.

Reply:
The issue is fixed by calling `.restore();` after each test, not by using sandboxes.

You could simply use `sinon.restore()` on the `afterEach`

Reply:
Good catch! That worked. Thanks for the suggestion.